tests/CMakeLists.txt

```diff
@@ -49,7 +50,7 @@ foreach(test IN LISTS tests)
         "cpp_${test}"
         FOLDER "C++ tests"
         SOURCES "${test}.cpp"
-        DEPENDENCIES cosim
+        DEPENDENCIES "cosim;libcbor::libcbor"
```

Comment:
Why does libcbor need to be listed here when none of our other dependencies are?

Reply:
Removed, not needed after update.